### PR TITLE
Allow arm encoder time to startup

### PR DESCRIPTION
**Pull request by OliverW10**
The arm encoder doesn't start returning correct values for a few seconds after startup causing the arm to freakout if enabled before that. This has been a problem a few times in testing but should not occur in normal match play, if the robot code restarts during a match however it is almost certain to happen so this adds a timeout before the arm starts doing anything.

### CODE REVIEW

File: components/arm.py [hunk 1]
```diff
@@ -47,6 +48,8 @@ class Arm:
     ROTATE_GRAVITY_FEEDFORWARDS = 2.5
 
     ARM_ENCODER_ANGLE_OFFSET = 0.325  # rotations 0-1
+    # time for the arm encoder to start working
+    ARM_STARTUP_TIME = 5
```
Comment (auscompgeek) [inline components/arm.py:52]: 5 seconds seems a bit pessimistic, how did we come up with this time? IIRC 3 seconds should be plenty of time for the RIO decoding to initialise.

**Review by auscompgeek — APPROVED** [review]
code LGTM

File: components/arm.py (hunk 1 above)
Comment (OliverW10) [reply components/arm.py:52]: 5 seconds was from my memory of the arm display when we deploy code, i think it was about 2-4 seconds before it started giving the correct value, i added a bit beacuse it freaking out in a match could cause serious damage and if the code is restarting an extra second or two wont matter much.

Comment (auscompgeek) [reply components/arm.py:52]: Fair enough, thanks for the explanation.

**Pull request merged**